src/player/Player.cpp

```diff
@@ -158,7 +162,7 @@ struct BaseUiPlane: Ui::Plane {
         stop{*this, {Ui::Snap::Right, play, ControlSize}, "Stop", Ui::Style::Danger},
         forward{*this, {Ui::Snap::Right, stop, HalfControlSize}, "»"},
         modelInfo{*this, {Ui::Snap::Top|Ui::Snap::Left, LabelSize}, "", Text::Alignment::LineLeft, 128, Ui::Style::Dim},
-        animationProgress{*this, {Ui::Snap::Right, forward, LabelSize}, "", Text::Alignment::LineLeft, 17}
+        animationProgress{*this, {Ui::Snap::Right, forward, LabelSize}, "", Text::Alignment::LineLeft, 18}
```

Comment:
No, this is my bug! :laughing:

Reply:
What do you mean? 😅

Reply:
> I guess I'll merge this after I fix the bug with animation progress being negative.

Ah

Reply:
Should be fixed with either e78edc7957ac6a4b972a36bd1594eb7abb6376de or ec63dd499ffbf05e89ee5a330f3e8faf58bbb28a that are currently in master. In particular, the animation time no longer goes negative for me, which triggered the assert here.

Can you rebase and confirm this is no longer needed? :)

Reply:
Works! ✔️